opm/parser/eclipse/EclipseState/Tables/MsfnTable.hpp

```diff
@@ -21,60 +21,33 @@
 #define	OPM_PARSER_MSFN_TABLE_HPP
 
 #include "SimpleTable.hpp"
+#include <opm/parser/eclipse/EclipseState/Tables/TableEnums.hpp>
 
 namespace Opm {
-    // forward declaration
-    class TableManager;
-
     class MsfnTable : public SimpleTable {
     public:
-
-        friend class TableManager;
-        MsfnTable() = default;
-
-        /*!
-         * \brief Read the PMISC keyword and provide some convenience
-         *        methods for it.
-         */
-        void init(Opm::DeckItemConstPtr item)
+        MsfnTable(Opm::DeckItemConstPtr item)
         {
-            SimpleTable::init(item,
-                             std::vector<std::string>{
-                                 "GasPhaseFraction",
-                                 "GasSolventRelpermMultiplier",
-                                 "OilRelpermMultiplier"
-                                     });
-
-            SimpleTable::checkNonDefaultable("GasPhaseFraction");
-            SimpleTable::checkMonotonic("GasPhaseFraction", /*isAscending=*/true);
-            SimpleTable::assertUnitRange("GasPhaseFraction");
-
-            SimpleTable::checkNonDefaultable("GasSolventRelpermMultiplier");
-            SimpleTable::checkMonotonic("GasSolventRelpermMultiplier", /*isAscending=*/true, /*isStriclyMonotonic=*/false);
-
-            SimpleTable::checkNonDefaultable("OilRelpermMultiplier");
-            SimpleTable::checkMonotonic("OilRelpermMultiplier", /*isAscending=*/false, /*isStriclyMonotonic=*/false);
-
+            m_schema = std::make_shared<TableSchema>( );
+            m_schema->addColumn( ColumnSchema( "GasPhaseFraction", Table::STRICTLY_INCREASING  , Table::DEFAULT_NONE));
```

Comment:
well, here you use
```c++
 ColumnSchema( "GasPhaseFraction" [...])
```

while above and below there are also instances of
```c++
 ColumnSchema( "GAS_PHASE_FRACTION" [...])
```

this problem is also in the JSON keyword definitions (they should be made consistent). The table code is not affected by the JSON though, because if you did not change this, integer indices are used to the column data of records.